Gemfile.local

```diff
@@ -1,5 +1,5 @@
 group :acceptance_testing do
   # Needed for podman testing
-  gem "docker-api", :git => 'https://github.com/trevor-vaughan/docker-api', :branch => 'podman-compat'
+  gem "docker-api", '~> 2.1'
```

Comment:
Wait, should this perhaps update it here instead?
```suggestion
  gem "docker-api", '~> 2.1'
```

Reply:
@ekohl Shoot..sorry, I missed this comment somehow until things refreshed :-/

Reply:
I'm not seeing the difference though

Reply:
If you do it here in Gemfile.local, it only benefits local checkouts. If you modify it in the gemspec it's done for any installation which guarantees that podman works. Otherwise someone may have pinned it to 2.0 and complain podman doesn't work.